lib/krikri/harvesters/marc_xml_harvester.rb

```diff
@@ -0,0 +1,106 @@
+module Krikri::Harvesters
+
+  class MarcXMLHarvester
+    include Krikri::Harvester
+
+    attr_reader :opts
+
+    def initialize(opts = {})
+      @max_records = opts.fetch(:max_records, :unlimited)
```

Comment:
I'm in favor of this being an option, though probably in `Krikri::Harvester` as @no-reply suggests.

It would be convenient to be able to limit an enqueued harvest.